app/models/registration.rb

```diff
@@ -521,4 +521,56 @@ def competition_events_changed?
   def serializable_hash(options = nil)
     super(DEFAULT_SERIALIZE_OPTIONS.merge(options || {}))
   end
+
+  def attempt_auto_accept
+    return false if Rails.env.production? && EnvConfig.WCA_LIVE_SITE?
+
+    failure_reason = auto_accept_failure_reason
+    if failure_reason.present?
+      log_auto_accept_failure(failure_reason)
+      return false
+    end
+
+    update_payload = build_auto_accept_payload
+    auto_accepted_registration = Registrations::RegistrationChecker.apply_payload(self, update_payload)
```

Comment:
Can you specify the cloning behavior explicitly here? Would make it easier if we ever decide to change the default value

Reply:
I don't follow this request - what is being cloned, and what default value might we change? 

The only thing I can think of is the contents of the update payload, but that's just abstracted in a very easy-to-reach method if we ever want to change it

Reply:
I mean that `apply_payload` has a `clone` parameter that you should explicitly specify, instead of relying on the implicit `false` default from `def self.apply_payload(registration, raw_payload, clone: true)`

Reply:
Makes sense! I have set `clone: false` explicitly